manifests/charts/istiod-remote/Chart.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+name: istiod-remote
+version: 1.1.0
+appVersion: 1.1.0
+tillerVersion: ">=2.7.2"
+description: Helm chart for istio control plane
+keywords:
+  - istio
+  - istiod-remote
```

Comment:
can you call this keyword `istio-remote`.

Reply:
see comment earlier